docs/general/polkadot-vision.md

```diff
@@ -0,0 +1,194 @@
+---
+id: polkadot-vision
+title: Polkadot Vision
+sidebar_label: Polkadot Vision
+description: Polkadot's Vision revealed by Gavin Wood at Decoded 2023.
+keywords: [polkadot vison, polkadot]
+slug: ../polkadot-vision
+---
+
+## A Multi-chain Future
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
+tens of thousands. Different projects have different value propositions, suggesting that the future
+will likely be multi-chain and that inter-chain communication will be crucial to establish
+collaborations and leveraging each other strengths.
```

Comment:
```suggestion
Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
tens of thousands. Different projects have different value propositions, suggesting that the future
will be multi-chain and that inter-chain communication will be crucial to establish
collaborations and leveraging each other strengths.
```